airflow/models/abstractoperator.py

```diff
@@ -397,11 +399,23 @@ def priority_weight_total(self) -> int:
         - WeightRule.DOWNSTREAM - adds priority weight of all downstream tasks
         - WeightRule.UPSTREAM - adds priority weight of all upstream tasks
         """
-        if self.weight_rule == WeightRule.ABSOLUTE:
+        from airflow.task.priority_strategy import (
+            _AbsolutePriorityWeightStrategy,
+            _DownstreamPriorityWeightStrategy,
+            _UpstreamPriorityWeightStrategy,
+        )
+
+        warnings.warn(
```

Comment:
Why are we deprecating this? Its odd to deprecate something with experimintal feature as if eventually we decide not to accept it as stable what would happen to this deprecation?

Reply:
You're right, I will remove it.